minitwit_nodejs/repositories/userRepository.js

```diff
@@ -35,13 +45,14 @@ function follow(whoID, whomID) {
  * @param {int} whomID
  */
 function unfollow(whoID, whomID) {
```

Comment:
I find this who/whom to be a bit misleading. What do you think?
```suggestion
function unfollow(followerD, followedID) {
```